jax/experimental/jax2tf/tests/shape_poly_test.py

```diff
@@ -1117,7 +1117,7 @@ def _make_harness(group_name: str, name: str,
                   check_result=True,
                   skip_jax_run=True,
                   tol=None,
-                  enable_and_diable_xla=False,
+                  enable_and_disable_xla=False,
                   expect_error=(None, None),
                   **params) -> Union[Harness, Sequence[Harness]]:
```

Comment:
Can this **params be removed? 

It seems like there's an over-zealous **params somewhere which allowed the typo to pass through undetected... Might be nice to look around and ensure that the 'leaf' functions don't have **params as an argument.